onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -108,20 +108,33 @@ Status GraphPartitioner::Partition(onnxruntime::Graph& graph) const {
             break;
           }
         }
-
         if (sub_graph_available_for_assignment) {
-          // Add fused node into <graph>
           std::string node_name = provider->Type() + "_" + capability->sub_graph->GetMetaDef()->name + "_" + std::to_string(count++);
           auto& fused_node = graph.FuseSubGraph(std::move(capability->sub_graph), node_name);
           fused_node.SetExecutionProviderType(provider->Type());
-          auto fused_kernel_func = capability->fuse_kernel_function;
-          if (fused_kernel_func != nullptr) {
-            // build the kernel definition on the fly, and register it to the fused_kernel_regisitry.
-            KernelDefBuilder builder;
-            BuildFusedKernelDef(builder, fused_node);
-            fused_kernel_registry->Register(builder, fused_kernel_func);
-          }
-        }
+          if (capability->need_compile)
+            nodes_need_compile.push_back(&fused_node);
+		}
+      }
+    }
+    if (nodes_need_compile.size() > 0) {
+      if (session_state.ExportDll()) {
```

Comment:
Can you add some comments for the logic here? It seems whether session_state.ExportDll() is true or false, provider always compiles the nodes. Where does the previously exported dll being loaded and registered then?

Reply:
Yes, during the graph partition phase, execution provider will always compile the node. the ExportDll() flag just indicate whether we export the compiled binary to dll or not. The flow about loading previously export dll is not in this PR, I will submit it later.
The flow should be lotus export the fused graph that contains a function node, the new session will load the fused graph, assign the function node to a function kernel, and delegate the compute to the functions in the exported dll.

Reply:
has add some comments and todo items for this part.